stories/Components/TableDocs.mdx

```diff
@@ -1,3 +1,5 @@
+`import { Table } from "@bigbinary/neetoui";`
```

Comment:
The name of this file can be more specific as it contains data for only 1 story, and move all the table related mdx to a folder